### PR TITLE
Deprecation

### PR DESCRIPTION
**Pull request by edewit**
removes uses of deprecated javascript invocation on the plugin

### CODE REVIEW

**Comment by edewit** [comment]
fixes issue [AGCORDOVA-92](https://issues.jboss.org/browse/AGCORDOVA-92)

File: www/aerogear-push.js [hunk 1]
```diff
@@ -30,6 +30,19 @@ function Push(){
     }
 }
 
+function notify() {
+    console.log('event fired');
```
Comment (lholmquist) [inline www/aerogear-push.js]: i would probably remove this console.log

**Comment by matzew** [comment]
:+1:

**Comment by abstractj** [comment]
@edewit @matzew I think this PR has some changes already addressed here https://github.com/aerogear/aerogear-cordova-push/pull/64, nope?

**Comment by matzew** [comment]
you mean the package.json file, right ? yeah, that looks very similar

On Tue, May 5, 2015 at 2:56 PM, Bruno Oliveira <notifications@github.com>
wrote:

> @edewit <https://github.com/edewit> @matzew <https://github.com/matzew> I
> think this PR has some changes already addressed here #64
> <https://github.com/aerogear/aerogear-cordova-push/pull/64>, nope?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-cordova-push/pull/65#issuecomment-99071537>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by abstractj** [comment]
@matzew correcten!

**Comment by edewit** [comment]
ooops seems I based the branch off of the wrong branch

**Comment by edewit** [comment]
@abstractj rebased

**Comment by abstractj** [comment]
@sebastienblanc @matzew I think you guys are free to push the button :black_square_button:

**Pull request merged**